crates/sled/src/lib.rs

```diff
@@ -5,11 +5,11 @@
 //! ```
 //! let t = sled::Tree::start_default("my_db").unwrap();
 //!
-//! t.set(b"yo!".to_vec(), b"v1".to_vec());
+//! t.set(b"yo!", b"v1".to_vec());
 //! assert!(t.get(b"yo!").unwrap().unwrap() == &*b"v1".to_vec());
 //!
 //! t.cas(
-//!     b"yo!".to_vec(),       // key
+//!     b"yo!",                // key
```

Comment:
I'm unreasonably excited about being able to write simple demos without having to litter .to_vec() etc... everywhere lol :)